package.json

```diff
@@ -1,56 +1,92 @@
 {
-  "name": "sorcery",
-  "description": "Resolve a chain of sourcemaps back to the original source",
-  "version": "0.10.0",
-  "author": "Rich Harris",
-  "repository": "https://github.com/Rich-Harris/sorcery",
-  "main": "dist/sorcery.cjs.js",
-  "jsnext:main": "dist/sorcery.es6.js",
+  "name": "sourcery-map",
+  "description": "Find your way through the chain of sourcemaps to the original source, like magic sourcery",
+  "version": "4.0.0-alpha.1",
+  "author": "Emmanuel Kimmerlin",
```

Comment:
This probably shouldn't be here :)

Reply:
Oups ! (French expression), did not know what I did, but will fix, sorry for that

Reply:
All good! Just wanted to point out the author and how it should be changed back to the original, not say that the PR had to be closed.